WalletWasabi/Blockchain/Analysis/BlockchainAnalyzer.cs

```diff
@@ -20,6 +20,8 @@ public BlockchainAnalyzer(int privacyLevelThreshold)
 	/// </summary>
 	public void Analyze(SmartTransaction tx)
 	{
+		bool isLikelyCoinjoin = tx.IsLikelyCoinjoin();
```

Comment:
Is there a reason to declare this variable here? Why not just check `if (!tx.IsLikelyCoinjoin())`

Reply:
Well now we only use it at one place, so true, it's leftover